ddtrace/ext/tags.go

```diff
@@ -63,6 +63,27 @@ const (
 	// ServiceName defines the Service name for this Span.
 	ServiceName = "service.name"
 
+	// UserEmail defines the user e-mail span tag
+	UserEmail = "usr.email"
```

Comment:
We have to change these to be OpenTelemetry compliant, otherwise we'll end up with duplicated tags: https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/span-general/#general-identity-attributes.

Why were these strings chosen to be different?

Reply:
Ok, I'd like to re-open the discussion to briefly understand the plan. Ultimately, we may want to switch all our tags to semantic ones, or provide support for both to avoid duplication of tags with OpenTelemetry.